node/tests/GlideClusterClient.test.ts

```diff
@@ -903,8 +904,9 @@ describe("GlideClusterClient", () => {
                                             expectedFlags,
                                         ),
                                 );
-                                functionStats =
-                                    await client.functionStats(route);
+                                functionStats = await client.functionStats({
+                                    route: route,
```

Comment:
add the decoder option to return binary data

Reply:
I need to make the test after modifying `Record< >` and `checkFunctionStatsResponse`